index.js

```diff
@@ -120,9 +137,25 @@ async function print(argv) {
     await page.goto(url, buildNavigationOptions(argv));
 
     console.error(`Writing ${argv.output || 'STDOUT'}`);
+
+    let height, width;
+    if (argv.magicformat) {
+        height = (await page.evaluate(
+            'Math.max(document.body.scrollHeight, document.body.offsetHeight, document.documentElement.clientHeight, document.documentElement.scrollHeight, document.documentElement.offsetHeight)'));
+        width = '1366';
```

Comment:
how was this value chosen? should we make it an option?

Reply:
Based on the most common browser resolution website designs focus on.
As there is no width option, we'd have to add a separate with.